doc.md

```diff
@@ -29,7 +29,13 @@ project_root
 [tool.pkgmt]
```

Comment:
we're now documenting things in the doc/ directory, please move things over there